k8s/eks/yaml/multus-daemonset.yml

```diff
@@ -0,0 +1,281 @@
+            networks. More information available at: https://github.com/k8snetworkplumbingwg/multi-net-spec'
+          type: object
+          properties:
+            apiVersion:
+              description: 'APIVersion defines the versioned schema of this represen
+                tation of an object. Servers should convert recognized schemas to the
+                latest internal value, and may reject unrecognized values. More info:
+                https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#resources'
+              type: string
+            kind:
+              description: 'Kind is a string value representing the REST resource this
+                object represents. Servers may infer this from the endpoint the client
+                submits requests to. Cannot be updated. In CamelCase. More info: https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#types-kinds'
+              type: string
+            metadata:
+              type: object
+            spec:
+              description: 'NetworkAttachmentDefinition spec defines the desired state of a network attachment'
+              type: object
+              properties:
+                config:
+                  description: 'NetworkAttachmentDefinition config is a JSON-formatted CNI configuration'
+                  type: string
+---
+kind: ClusterRole
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  name: multus
+rules:
+  - apiGroups: ["k8s.cni.cncf.io"]
+    resources:
+      - '*'
+    verbs:
+      - '*'
+  - apiGroups:
+      - ""
+    resources:
+      - pods
+      - pods/status
+    verbs:
+      - get
+      - update
+  - apiGroups:
+      - ""
+      - events.k8s.io
+    resources:
+      - events
+    verbs:
+      - create
+      - patch
+      - update
+---
+kind: ClusterRoleBinding
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  name: multus
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: ClusterRole
+  name: multus
+subjects:
+- kind: ServiceAccount
+  name: multus
+  namespace: kube-system
+---
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: multus
+  namespace: kube-system
+---
+kind: ConfigMap
+apiVersion: v1
+metadata:
+  name: multus-cni-config
+  namespace: kube-system
+  labels:
+    tier: node
+    app: multus
+data:
+  # NOTE: If you'd prefer to manually apply a configuration file, you may create one here.
+  # In the case you'd like to customize the Multus installation, you should change the arguments to the Multus pod
+  # change the "args" line below from
+  # - "--multus-conf-file=auto"
+  # to:
+  # "--multus-conf-file=/tmp/multus-conf/70-multus.conf"
+  # Additionally -- you should ensure that the name "70-multus.conf" is the alphabetically first name in the
+  # /etc/cni/net.d/ directory on each node, otherwise, it will not be used by the Kubelet.
```

Comment:
Explained [here](https://github.com/testground/testground/issues/1459).

Reply:
Let's link from the comment here too. On top of that, could you explain the comment about `70-multus.conf` a bit more? Is `7` of any significance? What ends up in `/etc/cni/net.d/` and how do I make sure all of those things are lexicographically greater than `70-multus.conf`?

Reply:
Added a comment in the yaml.

That comment about `70-multus.conf` was added by the people who developed multus; we left it as additional info.

Reply:
Did we copy that file from somewhere then? Could we add a link to the source?

Reply:
Yes, it was pulled from the original multus repo that can be found [here](https://github.com/k8snetworkplumbingwg/multus-cni/blob/master/deployments/multus-daemonset.yml).

Reply:
Which version or commit? Could you add a comment in the file with a link?
Please use a permalink not a link to `master`, that might change.

Reply:
This comment has been added to the file back in 2019. and has not changed since. [Permalink](https://github.com/k8snetworkplumbingwg/multus-cni/blob/ce11203f5c55f7ed6cc6278e9b4438e625e5c9b3/images/multus-daemonset.yml#L71).

Reply:
Could you add a link at the top of a file to https://github.com/k8snetworkplumbingwg/multus-cni/blob/ce11203f5c55f7ed6cc6278e9b4438e625e5c9b3/images/multus-daemonset.yml? The commit should be the same as we used when we copied that file.

Reply:
Yes, will be added as a comment.